README.md

```diff
@@ -49,7 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents] = ["Maurice", "Gaston"]
+town[:castle][:guests] = "Belle"
```

Comment:
This is valid but is there a method you could use to remove an element from an array rather than instantiating a new variable?